custom/opm/reports.py

```diff
@@ -1057,7 +1057,8 @@ def awc_data(self):
         # populate awcs with cases from that awc
         for case_object in case_objects:
             awc = case_object.awc_name
-            awcs[awc] = awcs.get(awc, []) + [case_object]
+            awc_uid = case_object.owner_id
+            awcs[awc_uid] = awcs.get(awc_uid, []) + [case_object]
```

Comment:
Yeah, may be not, because CMR, BPR splits pregnant cases under each AWC into separate rows anyway. (also see above comment)